pkg/sync/sync_file.go

```diff
@@ -577,6 +581,275 @@ func (sf *SyncingFile) waitForRestoreComplete() (err error) {
 	return nil
 }
 
+func (sf *SyncingFile) CloneToFileWithEncryption(sourceBackingImage, sourceBackingImageUUID string, encryption types.EncryptionType, credential map[string]string) (copied int64, err error) {
```

Comment:
NIT: Add a comment that explains the meaning of `encryption` for the source and the target

Reply:
added.